doc/docs/getting_started/multi_node.md

```diff
@@ -10,29 +11,32 @@ BlueChi is intended for multi-node environments with a predefined number of node
 The main node will be the **laptop**. So first of all, lets install the controller, the agent and the CLI tool as well as the SELinux policy on it:
 
 ```bash
-dnf install bluechi bluechi-agent bluechi-ctl bluechi-selinux 
+dnf install bluechi bluechi-agent bluechi-ctl bluechi-selinux
 ```
 
 The **raspberry pi** will be (one of) the managed node in this example. Therefore, only install the agent and the SELinux policy on it:
 
 ```bash
-dnf install bluechi-agent bluechi-selinux 
+dnf install bluechi-agent bluechi-selinux
 ```
 
 Once the installations succeeded, BlueChi needs to be configured on both machines.
 
 ### Configuring the main node
 
-Lets start with the main node (the **laptop** in this case). The configuration of the controller running on the **laptop** is similar to the [single node setup](#installation-and-configuration) with the exception of adding **pi** as another node to it:
+Lets start with the main node (the **laptop** in this case). The configuration of the controller running on the **laptop** is similar to the [single node setup](#installation-and-configuration) with two exceptions:
```

Comment:
"Lets" -> "Let's"

Reply:
Oh, this typo is happening a lot. Changed it also in other places. PTAL @mwperina